DiscordCode/FrontEnd/css/friend_page.css

```diff
@@ -0,0 +1,64 @@
+.friend_list {
```

Comment:
Looks good, may have to change some colors to provide overall consistency across the system.